perun-core/src/main/java/cz/metacentrum/perun/core/impl/ConsentsManagerImpl.java

```diff
@@ -13,10 +19,42 @@
  */
 public class ConsentsManagerImpl implements ConsentsManagerImplApi {
 
+	final static Logger log = LoggerFactory.getLogger(ConsentsManagerImpl.class);
+
 	private static JdbcPerunTemplate jdbc;
 
 	public ConsentsManagerImpl(DataSource perunPool) {
 		jdbc = new JdbcPerunTemplate(perunPool);
 		jdbc.setQueryTimeout(BeansUtils.getCoreConfig().getQueryTimeout());
 	}
+
+	@Override
+	public Consent createConsent(PerunSession perunSession, Consent consent) {
```

Comment:
We also need to create a new entry in the _consent_attr_defs_ table for each attribute the Consent object has.